packages/frontend/src/lib/BootcEmptyScreen.svelte

```diff
@@ -1,9 +1,109 @@
 <script lang="ts">
-import EmptyScreen from './upstream/EmptyScreen.svelte';
-import BootcIcon from './BootcIcon.svelte';
+import BootcSelkie from './BootcSelkie.svelte';
+import Link from './upstream/Link.svelte';
+import { faArrowCircleDown, faCube } from '@fortawesome/free-solid-svg-icons';
+import Button from './upstream/Button.svelte';
+import { onMount, tick } from 'svelte';
+import { bootcClient, rpcBrowser } from '../api/client';
+import { Messages } from '/@shared/src/messages/Messages';
+import { router } from 'tinro';
+
+let pullInProgress = false;
+let imageExists = false;
+let displayDisclaimer = false;
+let bootcAvailableImages: any[] = [];
+
+const exampleImage = 'quay.io/bootc-extension/httpd:latest';
+const bootcImageBuilderSite = 'https://github.com/osbuild/bootc-image-builder';
+const bootcSite = 'https://containers.github.io/bootc/';
+const centosBootcSite = 'https://github.com/CentOS/centos-bootc';
+const extensionSite = 'https://github.com/containers/podman-desktop-extension-bootc';
+
+async function gotoBuild(): Promise<void> {
+  router.goto('/build');
+}
+
+async function pullExampleImage() {
+  pullInProgress = true;
+  displayDisclaimer = false;
+  bootcClient.pullImage(exampleImage);
+
+  // After 5 seconds, check if pull is still in progress and display disclaimer if true
+  setTimeout(async () => {
+    if (pullInProgress) {
+      displayDisclaimer = true;
+      await tick(); // Ensure UI updates to reflect the new state
+    }
+  }, 5000);
+}
+
+onMount(async () => {
+  bootcAvailableImages = await bootcClient.listBootcImages();
+
+  return rpcBrowser.subscribe(Messages.MSG_IMAGE_PULL_UPDATE, async msg => {
+    console.log(msg);
+    if (msg.image === exampleImage) {
+      pullInProgress = !msg.success;
+      if (!pullInProgress) {
+        displayDisclaimer = false; // Ensure disclaimer is removed when not in progress
+      }
+    }
+    // Update the list of available images after a successful pull
+    if (msg.success) {
+      bootcAvailableImages = await bootcClient.listBootcImages();
+    }
+  });
+});
+
+// Each time bootcAvailableImages updates, check if 'quay.io/bootc-extension/httpd' is in RepoTags
+$: {
+  if (bootcAvailableImages && bootcAvailableImages.some(image => image.RepoTags.includes(exampleImage))) {
+    imageExists = true;
+  }
+}
 </script>
 
-<EmptyScreen
-  icon="{BootcIcon}"
-  title="No bootable container builds found"
-  message="Start your first build by clicking the 'Build' button" />
+<div class="flex w-full h-full justify-center items-center">
+  <div class="flex flex-col h-full items-center text-center space-y-3">
+    <!-- Bootable Container Icon -->
+    <div class="text-gray-700 py-2">
+      <svelte:component this="{BootcSelkie}" size="120" />
+    </div>
+
+    <h1 class="text-xl pb-4">Welcome to the Bootable Containers image builder!</h1>
```

Comment:
We're going to have more discussion on which terms to use after this PR, but for now I'd prefer to drop 'the' and 'image builder', i.e. "Welcome to Bootable Containers" and be the intro to bootable containers and not limit ourselves to only building images.